site/en/docs/extensions/reference/action/index.md

```diff
@@ -68,101 +63,233 @@ context.clearRect(0, 0, 16, 16);
+settings take priority over any global settings.
 
 ## Enabled state
 
-By default, toolbar actions are enabled (clickable) on every tab. You can
-control this using the `action.enable()` and `action.disable()` methods.
-This only affects whether the popup (if any) or `action.onClicked` event
-is dispatched to your extension; it does not affect the action's presence in the
-toolbar.
+By default, toolbar actions are enabled (clickable) on every tab. You can control this using the
+`action.enable()` and `action.disable()` methods. This only affects whether the popup (if any) or
+`action.onClicked` event is dispatched to your extension; it does not affect the action's presence
+in the toolbar.
+
+## Examples
+
+The following examples show some common for actions in an extension. For a more robust demonstation
+of action capabilities, see the [Action API example][sample-example] in the
+[chrome-extension-samples][repo-samples] repository.
+
+### Show a popup
+
+It's common for extensions to display a popup when the user clicks the extension's action. To
+implement this in your own extension, declare the popup in your `manifest.json` and specify the
+content Chrome should display in the popup.
+
+```json
+// manifest.json
+{
+  "name": "Action popup demo",
+  "version": "1.0",
+  "manifest_version": 3,
+  "action": {
+    "default_title": "Click to view a popup",
+    "default_popup": "popup.html"
+  }
+}
+```
+
+```html
+<!-- popup.html -->
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    html {
+      min-height: 5em;
+      min-width: 10em;
+      background: salmon;
+    }
+  </style>
+</head>
+<body>
+  <p>Hello, world!</p>
+</body>
+</html>
+```
+
+### Injecting a content script on click
+
+A common pattern in the extension ecosystem is for extensions to expose their primary functionality
+using the extension's action. This example is one such extension; when the user clicks the action,
+we will inject a content script into the current page and that content script will display an alert
+to verify that everything worked as expected.
+
+```json
+// manifest.json
+{
+  "name": "Action script injection demo",
+  "version": "1.0",
+  "manifest_version": 3,
+  "action": {
+    "default_title": "Click to show an alert",
+  },
+  "permissions": ["activeTab", "scripting"],
+  "background": {
+    "service_worker": "background.js"
+  }
+}
+```
+
+```js
+// background.js
+chrome.action.onClicked.addListener((tab) => {
+  chrome.scripting.executeScript({
+    target: {tabId: tab.id},
+    files: ['content.js']
+  });
+});
+```
+
+```js
+// content.js
+alert('Hello, world!');
+```
+
+### Emulating pageActions with declarativeContent
+
+The chrome.action API replaced the browserAction and pageAction APIs in Manifest V3. By default,
+actions are very similar to browser actions, but it is possible to emulate the behavior of a page
+action using the action API.
+
+This example shows how an extension author could implement background logic to (a) disable the
```

Comment:
Consider changing:
    an extension author could implement background logic to
to:
    an extension's background logic can

Reply:
Thanks for the suggestion. Applied.